mycodo/controllers/controller_trigger.py

```diff
@@ -133,15 +132,9 @@ def loop(self):
                 # Now only set PWM output
                 pwm_duty_cycle, ended = self.get_method_output(
                     self.trigger.unique_id_1)
-                self.logger.debug("trigger_run_pwm_method: {a} {b}".format(a=pwm_duty_cycle, b=ended))
                 if not ended:
                     self.timer_period += self.trigger.period
-                    output_channel = db_retrieve_table_daemon(OutputChannel).filter(
-                        OutputChannel.unique_id == self.trigger.unique_id_3).first()
-                    self.set_output_duty_cycle(
-                        self.trigger.unique_id_2,
-                        pwm_duty_cycle,
-                        output_channel=output_channel.channel)
+                    self.set_output_duty_cycle(pwm_duty_cycle)
```

Comment:
the db access was repeated multiple times and therefore moved into the set_output_duty_cycle method.